app/controllers/concerns/blacklight/suggest/suggest_helper.rb

```diff
@@ -0,0 +1,32 @@
+module Blacklight
+  module Suggest
+    module SuggestHelper
+      extend ActiveSupport::Concern
+      include Blacklight::SearchHelper
+
+      ##
+      # For now, only use the q parameter to create a
+      # Blacklight::Suggest::Response
+      # @param [Hash] params
+      # @return [Blacklight::Suggest::Response]
+      def suggestions(params)
+        request_params = { q: params[:q] }
+        Blacklight::Suggest::Response.new suggest_results(request_params), request_params, suggest_handler_path
```

Comment:
I wonder if some of this can use the `Blacklight::SearchBuilder` framework (so we could possibly support suggestions for e.g. Elasticsearch too)?